api-authorizations/src/main/scala-templates/default/com/normation/plugins/apiauthorizations/EnablePluginImpl.scala

```diff
@@ -38,6 +38,7 @@
 package com.normation.plugins.apiauthorizations
 
 import com.normation.plugins.PluginEnableImpl
+import com.normation.rudder.services.nodes.NodeInfoService
 
-final class CheckRudderPluginEnableImpl() extends PluginEnableImpl
+final class CheckRudderPluginEnableImpl(nodeInfoService: NodeInfoService) extends PluginEnableImpl
```

Comment:
Do you think in the future we will need to pass RudderConfig directly ? or even directly look into that object ?

Reply:
we could. As a general rule, I prefer to let service dependency in parameters, it's easier to test.